package.json

```diff
@@ -11,22 +11,28 @@
     "electron-packager": "8.5.1",
     "eslint-config-fluid": "1.0.0",
     "fluid-grunt-eslint": "18.1.2",
+    "gpii-survey-server": "git+https://github.com/danailbd/gpii-survey-server.git",
     "grunt": "1.0.1",
     "grunt-jsonlint": "1.1.0",
     "grunt-shell": "1.3.0",
+    "npm-run-all": "4.1.2",
     "shelljs": "0.3.0"
   },
   "scripts": {
     "start": "electron main.js",
-    "dev": "electron main.js ./configs app.dev",
+    "start-survey-server": "node ./node_modules/gpii-survey-server",
+    "start-psp-dev": "electron main.js ./configs app.dev",
+    "dev": "npm-run-all --parallel start-survey-server start-psp-dev",
     "test": "powershell provisioning/Tests.ps1 -originalBuildScriptPath './node_modules/gpii-windows/provisioning'; electron tests.js"
   },
   "dependencies": {
     "electron": "1.4.1",
     "electron-edge": "6.5.4",
     "gpii-windows": "git://github.com/danailbd/windows.git#hst-2017",
     "infusion": "3.0.0-dev.20170830T182157Z.392b2f8",
+    "json-rules-engine": "2.0.2",
```

Comment:
Normally when we introduce a significant new library into the GPII, we perform a technology evaluation exercise, which sets out the requirements on the library, surveys a good selection of those available and matches them against the criteria. You can see some examples of evaluations we have done in the past at https://wiki.gpii.net/w/Technology_Evaluation - . In this case, I'm not clear that the chosen technology solves the difficult part of the problem, which is scheduling of events to occur asynchronously when conditions are met in the future, without incurring too much polling. Note that if we are just considering evaluating rules or expressions encoded in a declarative form, the existing model transformation system described at https://docs.fluidproject.org/infusion/development/ModelTransformationAPI.html can do most of what is required, perhaps with the addition of a couple of new transformation rules. Also note that @kaspermarkus  implemented a somewhat similar system in the GPII's existing "ContextManager" at https://github.com/GPII/universal/blob/master/gpii/node_modules/contextManager/src/ContextManager.js - this maintains a "context" (very similar to your collection of "facts") which takes in environmental information, including the current time, etc. and through a polling system causes a change of "context" which is specified by declarative rules such as this one: https://github.com/GPII/universal/blob/master/gpii/node_modules/contextManager/test/js/ContextManagerUtilitiesTests.js#L51